src/test/java/com/wikia/webdriver/elements/communities/desktop/components/categories/PaginationControls.java

```diff
@@ -0,0 +1,88 @@
+package com.wikia.webdriver.elements.communities.desktop.components.categories;
+
+import com.wikia.webdriver.common.logging.Log;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class PaginationControls<T> extends BasePageObject {
+
+  private final T outer;
+  @FindBy(xpath = "//a[contains(text(),'First')]")
+  private WebElement firstButton;
+  @FindBy(css = ".category-page__pagination-next")
+  private WebElement nextButton;
+  @FindBy(css = ".category-page__pagination-prev")
+  private WebElement previousButton;
+  @FindBy(xpath = "//a[contains(text(),'Last')]")
+  private WebElement lastButton;
+
+  public PaginationControls (T outer) {
+    this.outer = outer;
+  }
+
+  //TODO:Check if it actually works
+  public boolean isFirstButtonIsVisible() {
+
+    boolean isVisible = isVisible(firstButton);
+//    Log.info("Next page button is visible.");
+
+    return isVisible;
+  }
+
+  //TODO:Check if it actually works
+  public boolean isNextButtonVisible() {
+
+    boolean isVisible = isVisible(nextButton);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "isVisible". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1488)